src/index.tsx

```diff
@@ -7,14 +7,19 @@ import React from "react";
 import ReactDOM from "react-dom";
 import { App } from "./Components/App/App";
 import * as serviceWorker from "./serviceWorker";
-
+import { FeatureToggleClient } from "FeatureToggleClient";
+import { AuthorizationClient } from "@itwinjs-sandbox";
 import "./index.scss";
 import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
 
-ReactDOM.render(
-  <App />,
-  document.getElementById("root"),
-);
+(async () => {
+  await FeatureToggleClient.initialize();
+  await AuthorizationClient.initializeOidc();
```

Comment:
OIDC init needs to be moved just before imodel starts loading. Token is not needed for gallery and editor display. Obtaining a token takes up to 1 second, which increase white screen delay before app load.

Reply:
Move OIDC init to Sample Visualizer component